### PR TITLE
Index ERC20 Bedenia 

### PR DESCRIPTION
**Pull request by nicefacer**

### CODE REVIEW

**Review by nicefacer — COMMENTED** [review]
Made the image 128x128

File: packages/indexer/src/config/network.ts [hunk 1]
```diff
@@ -312,6 +312,19 @@ export const getNetworkSettings = (): NetworkSettings => {
               decimals: 18,
             },
           ],
+          [
+            "0x55818Be03e5103e74f96dF7343Dd1862A6d215f2",
```
Comment (nofir) [inline packages/indexer/src/config/network.ts:316]: @nicefacer can you change the address to be lowercase?

**Review by nofir — CHANGES_REQUESTED** [review]

**Review by nicefacer — COMMENTED** [review]
Requested change done.

File: packages/indexer/src/config/network.ts (hunk 1 above)
Comment (nicefacer) [reply packages/indexer/src/config/network.ts]: requested change done

**Review by nofir — APPROVED** [review]

File: packages/indexer/src/config/network.ts [hunk 2]
```diff
@@ -312,6 +312,19 @@ export const getNetworkSettings = (): NetworkSettings => {
               decimals: 18,
             },
           ],
+          [
+            "0x55818be03e5103e74f96df7343dd1862a6d215f2", 
+            {
+              contract: "0x55818be03e5103e74f96df7343dd1862a6d215f2",
+              name: "BIDENIA",
+              symbol: "BIE",
+              decimals: 8,
```
Comment (nofir) [inline packages/indexer/src/config/network.ts:321]: Screenshot 2023-09-20 at 12 44 35 PM

Just confirming the number of decimals. otherwise looks ok

**Review by nofir — CHANGES_REQUESTED** [review]

File: packages/indexer/src/config/network.ts (hunk 2 above)
Comment (nicefacer) [reply packages/indexer/src/config/network.ts:321]: The BIdenia decimal is 8.


The image you shared is for Biden Token not BIdenia.

**Pull request merged**